docs/HOWTO/90_Misc/Unpack_dvd.rst

```diff
@@ -1,28 +1,43 @@
 .. _howto-unpack-dvd:
 
-HOWTO Extract the Full OS RPM Package Set
-=========================================
+HOWTO Use unpack_dvd to Extract the Full OS RPM Package Set
+===========================================================
 
 The SIMP ISO provides a minimal set of packages.
 
 If you require additional OS packages, you can extract them from vendor ISOs using
-:program:`unpack_dvd`.
+:program:`unpack_dvd`.  Additionally you can extract the extract the PXE boot
+TFTP boot files from the ISO to the rsync directories.
+
+Use option :code:`unpack_dvd --help` to see all options available for :program:`unpack_dvd`.
+
+Extract the OS Packages
+-----------------------
 
 :program:`Unpack_dvd` extracts the OS RPMs to :file:`/var/www/yum/<OperatingSystem>`
 in a directory named after the OS version it determines from files on the ISO.
 It then creates a link to the major version.  If version 7.6 is extracted,
 7 will be linked to 7.6.
 
+After extracting all the RPMs it will create or update a repository in the
+versioned directory.
+
 .. NOTE::
 
    If unpack_dvd can only determine the major OS version from the files
    on the ISO, :program:`unpack_dvd` will ask you to supply a more descriptive
    version number using ``-v`` option.
 
+The following example will:
+
+* extract the RPMs to :file:`/var/www/yum/CentOS/7.8.2003`
```

Comment:
When I ran the program the files were root:root 0644.  If they should be root:apache 0640, the user will have to adjust.

Reply:
I found that but it is an error introduced.  I am looking to see if I can fix it.

Reply:
It does change the permissions.  I changes the permissions only of the repos which is all they need.  I made some notes in the HOWTO about permissions.